docs/configuration.asciidoc

```diff
@@ -108,6 +108,26 @@ NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+
 | Name of the entry assembly       | String 
 |============
 
+[float]
+[[config-service-version]]
+==== `ServiceVersion`
+
+This is used to know exact version of your service.
```

Comment:
Since this configuration option is common to all Elastic APM agents I recommend copying documentation from some other agent - for example from [Java agent](https://www.elastic.co/guide/en/apm/agent/java/current/config-core.html#config-service-version).

Reply:
Will fix it

Reply:
will fix it